eve/main.yml

```diff
@@ -5,10 +5,20 @@ branches:
     stage: "pre-merge"
 
 models:
+  - Git: &git_pull
+      name: git pull
+      repourl: "%(prop:git_reference)s"
+      mode: full
+      method: clobber
+      retryFetch: true
+      haltOnFailure: true
   - Upload: &upload_artifacts
       source: tests/artifacts
       urls:
         - "*"
+  - SetProperty: &docker_image_name
+      property: docker_image_name
+      value: '%(secret:private_registry_url)s/zenko/zenko-e2e'
```

Comment:
the secret value will be stored in the build properties, a visible by all in the build tabs.